### PR TITLE
added logic for reaching articles with comments, added test for keys …

### PR DESCRIPTION
**Pull request by pippaamy**
…and a test for descending order

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ app.patch("/api/articles/:article_id", patchArticleById)
 
 app.get("/api/users", getUsers)
 
+app.get("/api/articles", getArticle)
```
Comment (dappernerddesigns) [inline app/app.js:18]: Very picky I know, but you've got an end point of articles (plural) and a controller of getArticle (singular). Should this be getArticles?

**Review by dappernerddesigns — APPROVED** [review]
Nicely done, some super picky bits though, you've used singular names on getArticle and selectArticle. Both of these are returning several things and should really be pluralised :D

File: app/app.js (hunk 1 above)
Comment (pippaamy) [reply app/app.js:18]: will change that now

Comment (pippaamy) [reply app/app.js]: thanks!

**Pull request merged**